static/js/compose.js

```diff
@@ -732,6 +732,11 @@ export function initialize() {
         compose_actions.update_placeholder_text();
     });
 
+    $("#compose").on("click", ".to_text", (e) => {
+        e.preventDefault();
+        narrow.to_compose_target();
+    });
```

Comment:
This should be adjacent to the existing handler for the stream message version of this.

Reply:
Removed this altogether, sharing a class means they can share the same call to listen for the click event